blog/2025-09-30-conda-roadmap-q4-2025.mdx

```diff
@@ -0,0 +1,82 @@
+
+**What we are working on:** We are implementing support for **sharded repodata**, as proposed and accepted in [Conda Enhancement Proposal (CEP) 16](https://conda.org/learn/ceps/cep-0016). Instead of fetching one massive file, conda will be able to download smaller, targeted shards of metadata.
+
+**How you will benefit:** Package installs and updates will complete faster, conda will feel more responsive in daily use, and automated workflows like CI will run more efficiently with reduced bandwidth costs.
+
+---
+
+### Better Integration with the PyPI Ecosystem
+
+**What we heard:** Sometimes the package you need exists only on PyPI. Many users fall back on `pip install` inside a conda environment, but this approach often leads to dependency conflicts or broken environments that can take hours to fix.
+
+**What we are working on:** We are working on better integration with the PyPI ecosystem through the conda-pypi plugin. This work focuses on providing a safer way to use conda to access and install pure Python packages distributed as wheels within conda environments, while laying the foundation for broader and more native integration in the future.
+
+**How you will benefit:** This plugin will offer a safer path than mixing pip and conda directly, reducing environment breakage and making it easier to try packages that are not yet available on conda channels.
+
+---
+
+## Looking Ahead
+
+We will continue posting roadmap updates each quarter and will expand the horizon to cover 6–12 months as our planning process matures. Our goal is to make conda development more predictable and transparent. Each update will show progress on current priorities and preview what is next, shaped by your feedback and contributions.
+
+You can track progress live on the [GitHub roadmap board](https://github.com/orgs/conda/projects/22), where you will find current status, upcoming milestones, and discussions on specific features.
+
+---
+
+## Shape Conda’s Future With Us
+
+Conda is a community project, and these roadmap priorities reflect your input. Here are ways to influence what comes next:
+
+- **Share your feedback:** File [GitHub issues](https://github.com/conda/conda/issues) when conda does not work as expected or you have an idea for improvement
+- **Join the conversation:** [Conda community calls](https://conda.org/community/calendar) are where we discuss priorities, propose solutions, and decide what matters most
+- **Connect with maintainers:** Find us on [Zulip](https://conda.zulipchat.com/) and the [Discourse forum](https://conda.discourse.group/) to troubleshoot, suggest features, or just learn more about our thinking
+- **Take the [Python packaging survey](https://anaconda.surveymonkey.com/r/py-package-2025):** Every response helps us better understand your needs
+- **[Contribute to conda](https://github.com/conda/conda/blob/main/CONTRIBUTING.md)** or even become a maintainer yourself!
```

Comment:
```suggestion
- **[Contribute to conda](https://docs.conda.io/projects/conda/en/latest/dev-guide/contributing.html)** or even become a maintainer yourself!
```

Let's drive traffic to our docs site instead of GitHub 😉.